### DISCUSSION TITLE
Pass value into SQL from SPARQL?

### DISCUSSION THREAD

**Discussion by jyh0710** [ontop/ontop#588] [Q&A]
Hi,

We have a VKG stack setup with OnTop, Denodo, and a web service that requires JWT for IdAM.  We are wondering if we can pass values (e.g. the JWT string) from SPARQL - passthrough OnTop mapping - into SQL directly (so Denodo can take and re-interpret it)?  Thanks!

Yong

**Comment by bcogrel** [comment]
Hi @jyh0710, thanks for you interesting question.

At the moment, Ontop does not support this case, because the configuration of the DB connection is not done once when starting the endpoint and is the same for all users.

Making the DB connection dynamic using tokens coming from the HTTP headers would require some refactoring, but that's definitely an interesting idea!

Best,
Benjamin
